02_configure_host.sh

```diff
@@ -36,6 +36,23 @@ if [ ! -z "${VM_NODES_FILE}" ]; then
   exit 1
 fi
 
+# If deploying to a real baremetal environment, we need to make
+# sure all nodes are powered down
+if [[ "${NODES_PLATFORM}" == "baremetal" ]] && [ -n "$NODES_FILE" ] ; then
+    set +x
+    cat $NODES_FILE | jq -r '.nodes[].driver_info | ( .address + " " + .username + " " + .password ) ' |
+    while read ADDRESS USER PASSWORD ; do
+        if [[ $ADDRESS =~ ^ipmi://(.*)$ ]] ; then
+            IPMIIP=${BASH_REMATCH[1]}
+            ipmitool -I lanplus -H $IPMIIP -U $USER -P $PASSWORD power off || echo "WARNING: Skipping power down of $ADDRESS"
```

Comment:
Maybe in case of error it could be useful to print also the error code

Reply:
done